.moban.yaml

```diff
@@ -4,7 +4,7 @@ name: corobo
 
 dependencies:
   - pyopenssl<17.5
```

Comment:
please investigate why we had this pin.  and try to remove it.
may be it isnt needed any more

Reply:
okay, I'll look into it.

Reply:
https://github.com/coala/corobo/pull/592

Reply:
Remove it entirely in this PR